cluster-scope/overlays/prod/moc/infra/externalsecrets/rook-ceph-external-cluster-details.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: rook-ceph-external-cluster-details
+  namespace: openshift-storage
+spec:
+  secretStoreRef:
+    name: opf-vault-store
+    kind: SecretStore
```

Comment:
this secret store needs to be added for this namespace, and allow vault infra SA to access vault secrets for moc-infra
docs on how to do this: https://www.operate-first.cloud/apps/content/vault_eso/add_external_secrets_to_ns.html#enable-namespace-to-fetch-secrets-from-vault

Reply:
Resolved, I've totally missed that, thank you!